.babelrc

```diff
@@ -0,0 +1 @@
+{"presets": ["env"]}
```

Comment:
This is weird. I just spent 2 hours trying to figure why my Webpack wasn't correctly transorming one the the peer dependancies during the test (vue-masony that uses the `const` keyword which is not understood by PhantomJS). The only way to solve the problem was by adding this file. This should not happen as the Babel configuration is already present in Webpack. I opened an issue upstream.